### PR TITLE
Patterns - links

### PR DESCRIPTION
**Pull request by mhewson**
Link doc updates

### CODE REVIEW

File: docs_src/assets/css/src/docs.css [hunk 1]
```diff
@@ -715,7 +715,7 @@ h1[id] {
   margin: 0;
   border: 0.1rem solid #eee;
   border-left-width: 0.5rem;
-  border-radius: 0.3rem;
+  border-radius: -0.3rem;
```
Comment (cowills) [inline docs_src/assets/css/src/docs.css:718]: are these negative border radii correct?

File: docs_src/elements/typography.html [hunk 2]
```diff
@@ -2,7 +2,8 @@
 layout: default
 title: Typography
 slug: typography
-lead: "An overview of Typography: headings/families, labels, and much more."
+lead: "Text container used for any text element such as paragraphs, headings, and lists. Establishes vertical spacing and
+optimizes the reading experience"
```
Comment (cowills) [inline docs_src/elements/typography.html:6]: is this line-break intentional? not sure if it affects styling at all

File: docs_src/assets/css/src/docs.css (hunk 1 above)
Comment (mhewson) [reply docs_src/assets/css/src/docs.css:718]: yea the heading sizes changed and moved the icon alignment down with it

File: docs_src/elements/typography.html (hunk 2 above)
Comment (mhewson) [reply docs_src/elements/typography.html:6]: yea intended - no it does not effect the display

File: docs_src/assets/css/src/docs.css (hunk 1 above)
Comment (cowills) [reply docs_src/assets/css/src/docs.css:718]: but is a negative border-radius even something that works?
https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius seems to indicate that negative values are invalid

**Pull request merged**